### DISCUSSION TITLE
TP across bungee

### DISCUSSION THREAD

**Discussion by ScottGB** [Zrips/CMI#3947] [Feature Requests]
**Description of feature/improvement request:**
Be able to teleport to a person who's currently online, but on a different part of the server network.

---
### Detailed explanation of what it should do, when and in what situation: ###
If someone is online, you can type /tp (or /tpa) to teleport to them, and if they're on another part of the network (say you're in survival and they are in creative), it will be able to switch your server and teleport all in one!

If you want to teleport to their location in the server you are on, maybe add a modifier like "-l" (for -local)

### Basic explanation as to why this feature is needed and/or useful to have: ###
I'm quite lazy, and a streamlined system like this would be really nice

**Comment by Barbadosian** [comment]
This is a much wanted feature!
